IntroToCSharp/Pages/Activities/SettingBuilder.razor

```diff
@@ -0,0 +1,212 @@
+        <MudListItem Icon="@Icons.Filled.Circle">Practice working with <Code>integers</Code>.</MudListItem>
+        <MudListItem Icon="@Icons.Filled.Circle">Practice changing the color of the terminal.</MudListItem>
+    </MudList>
+</Section>
+
+<Section Title="Where to start?">
+    <Text>
+        When you think of many of your favorite games what comes to mind? Perhaps you think of an astronaut
+        flying through an astreroid field in space like in No Man's Sky. Or maybe you think of a plumber jumping
+        on evil turtles to Store a princess like Mario. Maybe it is even something scarier where you are trapped
+        in a Kid's Pizza store with animatronics trying to come to life and get you!
+    </Text>
+    <Text>
+        There are so many different settings that games or stories might take place but the first thing you should figure out
+        is what kind of "genre" the setting is in. Genre is a term that generally means the particular "style" and
+        "world" of the game or story. By knowing your genre you know what places <i>could</i> be possible and what places 
+        wouldn't be possible. you wouldn't expect to see a space level in a fantasy game like Skyrim!
+    </Text>
+    <Text>
+        To help the user figure out what genre the setting is in, let's give our users a list of options to help them
+        generate their own ideas for a genre. You can use the following list as a base but feel free to add more if you want!
+    </Text>
+    <Note>
+        <b>Extra Challenge!</b><br>
+        Can you figure out how to add bullet points to your list as youll? Hint: You should check out the <b>@AsciiArt.PageRef.Name</b> lesson
+        for a possible value that might work!
+    </Note>
+    <MudList>
+        <MudListItem Avatar="@Icons.Filled.MenuBook">Fantasy</MudListItem>
+        <MudListItem Avatar="@Icons.Filled.MenuBook">Horror</MudListItem>
+        <MudListItem Avatar="@Icons.Filled.MenuBook">Adventure</MudListItem>
+        <MudListItem Avatar="@Icons.Filled.MenuBook">Sci-fi</MudListItem>
+        <MudListItem Avatar="@Icons.Filled.MenuBook">Post-Apocalyptic</MudListItem>
+        <MudListItem Avatar="@Icons.Filled.MenuBook">Mystery</MudListItem>
+    </MudList>
+</Section>
+    <TaskItem Name="Create Project">Create a project called "Setting Builder"</TaskItem>
+    <TaskItem Name="Create Introduction">Create an introduction to tell the user what your program is.</TaskItem>
+    <TaskItem Name="Ask for Genre">Ask the user for a genre input and then Display a list of genres to help them out.</TaskItem>
+    <TaskItem Name="Store user Input">Store the users input as "genreInput".</TaskItem>
+
+<Section Title="Somewhere to call home.">
+    <Text>
+        Great job! You have a start to your setting builder! Now you can get into some fun by building your first in your 
+        Setting Builder. We'll need to think about the details of our program as if we were building our own setting.
```

Comment:
Proof read and revise: "Now you can get into some fun by building your first in your Setting Builder."